packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -202,14 +202,16 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         }
 
         // fired once the spec bridge is set up and ready to receive messages
-        communicator.once('bridge:ready', (_data, specBridgeOriginPolicy) => {
+        communicator.once('bridge:ready', async (_data, specBridgeOriginPolicy) => {
           if (specBridgeOriginPolicy === originPolicy) {
             // now that the spec bridge is ready, instantiate Cypress with the current app config and environment variables for initial sync when creating the instance
             communicator.toSpecBridge(originPolicy, 'initialize:cypress', {
               config: preprocessConfig(Cypress.config()),
               env: preprocessEnv(Cypress.env()),
             })
 
+            await Cypress.backend('cross:origin:bridge:ready', { originPolicy })
```

Comment:
Are there any performance implications here?

Reply:
Just the time it takes to contact the server to add a new origin to the remote states.